quinn/src/endpoint.rs

```diff
@@ -482,6 +493,9 @@ impl State {
                                         .send(ConnectionEvent::Proto(event));
                                 }
                                 Some(DatagramEvent::Response(transmit)) => {
+                                    if !primary_socket {
+                                        continue;
```

Comment:
It's not obvious to me why this should happen -- maybe add a comment?

Reply:
I tried to avoid doing anything non-essential on the abandoned path.  But on second though, it was unnecessarily cautious.  `DatagramEvent::NewConnection` cannot arrive to the client on the abandoned path during an client migration.  And to tell the truth, I don't understand when a `DatagramEvent::Response` is generated. 

In the latest version, I removed these tests.